integrations/apollo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nhost/apollo",
-  "version": "4.13.4",
+  "version": "5.0.0",
```

Comment:
[5.0.0](https://www.npmjs.com/package/@nhost/react-apollo/v/5.0.0) has been published by accident a year ago and is deprecated. We must skip this version before publishing - either bumping to `6.0.0` or `5.0.1`

Reply:
I vote for 5.0.1

Reply:
done. it will have to be done again if there's another commit on the `main` branch before we merge this pull request

Reply:
btw I did not point the right package - the problem is on `@nhost/react-apollo`

Reply:
I don't plan to merge anything until this is not released, so we should be good.